src/commonTest/kotlin/com/charleskorn/kaml/YamlListTest.kt

```diff
@@ -75,6 +76,29 @@ class YamlListTest : DescribeSpec({
             }
         }
 
+        describe("getting elements from a list") {
+            val firstItemPath = YamlPath.root.withListEntry(0, Location(4, 5))
+            val secondItemPath = YamlPath.root.withListEntry(1, Location(6, 7))
+
+            val list = YamlList(
+                listOf(
+                    YamlScalar("item 1", firstItemPath),
+                    YamlScalar("item 2", secondItemPath)
+                ),
+                YamlPath.root
+            )
+
+            it("returns element") {
```

Comment:
For these two tests, it would be good to describe the situation in addition to the desired behaviour (eg. "if the index is within the bounds of the list's size" or "if the index is outside the bounds of the list's size").